charts/stable/paperless-ng/values.yaml

```diff
@@ -25,8 +59,24 @@ persistence:
   config:
     enabled: true
     mountPath: "/config"
-  data:
+  consume:
     enabled: true
-    mountPath: "/data"
-  varrun:
+    mountPath: "/consume"
+  static:
     enabled: true
+    mountPath: "/static"
+  media:
+    enabled: true
+    mountPath: "/media"
+
+redis:
+  enabled: true
+  existingSecret: "rediscreds"
+  redisUsername: paperless-ng
+  redisDatabase: "0"
```

Comment:
why add this?

Reply:
I'm trying to use the `url` key from redisinjector, which needs to have the mentioned values. 
Am I still not getting it? :/

`url: {{ ( printf "redis://%v:%v@%v-redis:5432/%v" .Values.redis.redisUsername $dbPass .Release.Name .Values.redis.redisDatabase  ) | b64enc | quote }}`

Reply:
the redis injector just creates a secret, what does that have to do with you selectin database 0?
Database 0 is already the default and it's set nowhere else, so I just wonder why you decided to add it here.

Reply:
Aka: please get rid if this redundant line.